src/test/resources/data/published_requests/outgoing_mocked_request1_missing_sample_status.json

```diff
@@ -0,0 +1,354 @@
+  "investigatorName": "John Smith",
+  "investigatorEmail": "smithj@mskcc.org",
+  "dataAnalystName": "Poin Dexter",
+  "dataAnalystEmail": "dexterp@mskcc.org",
+  "otherContactEmails": "dexterp@mskcc.org",
+  "dataAccessEmails": "",
+  "qcAccessEmails": "",
+  "isCmoRequest": true,
+  "bicAnalysis": false,
+  "samples": [
+    {
+      "smileSampleId": "afe750e6-8756-11eb-81b0-acde48001122",
+      "smilePatientId": "6cc9f242-875a-11eb-ae91-acde48001122",
+      "cmoSampleName": "C-MP789JR-X001-d",
+      "sampleName": "XXX002_P3_12345_L1",
+      "sampleType": "Primary",
+      "oncotreeCode": "CLL",
+      "collectionYear": "",
+      "tubeId": "",
+      "libraries": [
+        {
+          "barcodeId": "IDT29",
+          "barcodeIndex": "ATTGAGGA",
+          "libraryIgoId": "MOCKREQUEST1_B_1_1_1_1",
+          "libraryVolume": 35.0,
+          "libraryConcentrationNgul": 34.8,
+          "captureConcentrationNm": "11.49425287356322",
+          "captureInputNg": "400.0",
+          "captureName": "Pool-MOCKREQUEST1_B-Tube7_1",
+          "runs": [
+            {
+              "runMode": "HiSeq High Output",
+              "runId": "RUNID_0123",
+              "flowCellId": "X5KL2KKAY",
+              "readLength": "",
+              "runDate": "2018-06-05",
+              "flowCellLanes": [
+                5
+              ],
+              "fastqs": [
+                "/FASTQ/Project_MOCKREQUEST1_B/Sample_XXX002_P3_12345_L1_IGO_MOCKREQUEST1_B_1/XXX002_P3_12345_L1_IGO_MOCKREQUEST1_B_1_S82_R1_001.fastq.gz",
+                "/FASTQ/Project_MOCKREQUEST1_B/Sample_XXX002_P3_12345_L1_IGO_MOCKREQUEST1_B_1/XXX002_P3_12345_L1_IGO_MOCKREQUEST1_B_1_S82_R2_001.fastq.gz"
+              ]
+            }
+          ]
+        }
+      ],
+      "cmoPatientId": "C-MP789JR",
+      "primaryId": "MOCKREQUEST1_B_1",
+      "investigatorSampleId": "XXX002_P3_12345_L1",
+      "species": "Human",
+      "sex": "F",
+      "tumorOrNormal": "Tumor",
+      "preservation": "Frozen",
+      "sampleClass": "PDX",
+      "sampleOrigin": "Tissue",
+      "tissueLocation": "",
+      "baitSet": "GENESET101_BAITS",
+      "genePanel": "GENESET101_BAITS",
+      "datasource": "igo",
+      "igoComplete": true,
+      "cmoSampleIdFields": {
+        "naToExtract": "",
+        "sampleType": "Tissue",
+        "normalizedPatientId": "MRN_REDACTED",
+        "recipe": "GENESET101_BAITS"
+      },
+      "patientAliases": [
+        {
+          "namespace": "cmo",
+          "value": "C-MP789JR"
+        }
+      ],
+      "sampleAliases": [
+        {
+          "namespace": "investigatorId",
+          "value": "XXX002_P3_12345_L1"
+        },
+        {
+          "namespace": "igoId",
+          "value": "MOCKREQUEST1_B_1"
+        }
+      ],
+      "additionalProperties": {
+        "isCmoSample": "true",
+        "igoRequestId": "MOCKREQUEST1_B"
+      },
+      "status": [
+        {
+        "validationStatus": true,
+        "validationReport": ""
```

Comment:
I'd be interested in seeing if it can detect changes even when validation report isn't an empty string (and not relying on the different being in the value of `validationStatus` - can you make one validation report as the following:

```
"validationReport" : [
    { "sampleOrigin: "Other"},
    { "sampleClass: "Unknown"}
]
```

Reply:
It'd make more sense to put this together with the mocked sample status you had set as `false`